zip-0032.html

```diff
@@ -1158,4 +1158,4 @@
         </section>
     </section>
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Not sure why these changed 🤦 but I guess they should have newlines at the end anyway